open-weather-connector-test/src_test/com/axonivy/connector/openweather/test/it/BaseProcessTest.java

```diff
@@ -0,0 +1,74 @@
+package com.axonivy.connector.openweather.test.it;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.BeforeEach;
+
+import ch.ivyteam.ivy.application.IApplication;
+import ch.ivyteam.ivy.bpm.engine.client.BpmClient;
+import ch.ivyteam.ivy.bpm.engine.client.element.BpmProcess;
+import ch.ivyteam.ivy.bpm.engine.client.sub.SubRequestBuilder;
+import ch.ivyteam.ivy.bpm.exec.client.IvyProcessTest;
+import ch.ivyteam.ivy.environment.AppFixture;
+import ch.ivyteam.ivy.rest.client.RestClient;
+import ch.ivyteam.ivy.rest.client.RestClients;
+
+@IvyProcessTest(enableWebServer = true)
+public class BaseProcessTest {
+	private static final String APP_ID_KEY = "appId";
+	private static final String APP_ID_PROP_KEY = "AUTH." + APP_ID_KEY;
+	private static final String WEATHER_DATA_URL_KEY = "weatherDataUrl";
+	private static final String WEATHER_GEO_URL_KEY = "weatherGeoUrl";
+	private static final String LOCAL_CREDENTIALS_FILE_PATH = "credentials.properties";
+	private static final String WEATHER_DATA_REST_CLIENT_NAME = "WeatherData (Openweathermap weather API)";
+	private static final String GEO_DATA_REST_CLIENT_NAME = "GeocodingCoordinates (Openweathermap geocoding API)";
+	protected static final String RESULT_KEY = "result";
+	protected static final String RESULTS_KEY = "results";
+	protected static final Double TEST_LON_VALUE = 40.7484;
+	protected static final Double TEST_LAT_VALUE = -73.9967;
+	protected static final String TEST_ZIPCODE_VALUE = "10001";
```

Comment:
I think this class is about setting up the client ... and should not be mixed up with assertions ... I'd rather move this out into a distinct interface.